docs/examples.md

```diff
@@ -304,6 +304,11 @@ Common configuration properties that you might want to change from their default
 - [cache.max.bytes.buffering](https://kafka.apache.org/documentation/#streamsconfigs):
   The default value in KSQL is `10000000` (~ 10 MB);
 
+- fail.on.deserialization.error:
+  When set to true (the default), any errors that occur when deserializing a record will result in the the error being logged and the record being dropped.
+  If you wish to halt processing on deserialization errors you should set this to false.
```

Comment:
seems a bit counter-intuitive ? i.e. set the "fail on error" flag to false to NOT halt ("fail") on error ? i predict significant user confusion for this...

Reply:
doh! thanks for pointing out. It is the other way around!

Reply:
lol - makes much more sense now!